### ISSUE TITLE
[webgpu] Add batchNorm kernel.

### ISSUE THREAD

**Issue by annxingyuan** [tensorflow/tfjs#2358]
Used in handtracking.
